exonum-java-binding-cryptocurrency-demo/src/test/resources/service-resources/responses/getActualConfiguration_response.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This file does not seem to belong to the cryptocurrency-demo?

Reply:
I'm opened for suggestions